### PR TITLE
hledger: add page

### PR DESCRIPTION
**Pull request by gutjuri**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution. 

> Even though there are smaller flags in the manual, I think the current ones are good as they are descriptive.

File: pages/common/hledger.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# hledger
+
+> A plain text accounting software for the command line.
```
Comment (navarroaxel) [inline pages/common/hledger.md:3]:
```suggestion
> A plain text accounting software for the command-line.
```

File: pages/common/hledger.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+> More information: <https://hledger.org>.
+
+- Use the interactive `add` command to add transactions to your journal:
```
Comment (navarroaxel) [inline pages/common/hledger.md:6]: Perhaps this better
```suggestion
- Add transactions to your journal interactively:
```

File: pages/common/hledger.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`hledger add`
+
+- Show the account hierarchy, using a journal file in a non-standard location:
```
Comment (navarroaxel) [inline pages/common/hledger.md:10]: Perhaps this better
```suggestion
- Show the account hierarchy, using a specific journal file:
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**